hail/python/hailtop/aiogoogle/client/compute_client.py

```diff
@@ -17,9 +21,15 @@ async def request_with_wait_for_done(request_f, path, params: MutableMapping[str
 
     delay = 0.2
     while True:
-        resp = await request_f(path, params=params, **kwargs)
-        if resp['status'] == 'DONE':
-            return resp
+        try:
+            resp = await request_f(path, params=params, **kwargs)
+            if resp['status'] == 'DONE':
+                return resp
+        except aiohttp.ClientResponseError as e:
+            if e.status == 403:
```

Comment:
I think we have to also check the message for the string "Rate Limit Exceeded". 403 could also be a real permissions issue.

I think we should just add this to is_transient_errors, it could show up in other APIs and the response is always the same: exponentially back off.